namespaces/cloud-platform-live-0.k8s.integration.dsd.io/adrian-sample-app-dev/resources/ecr.tf

```diff
@@ -0,0 +1,33 @@
+terraform {
+  backend "s3" {}
+}
+
+provider "aws" {
+  region = "eu-west-1"
+}
+
+/*
+ * Make sure that you use the latest version of the module by changing the
+ * `ref=` value in the `source` attribute to the latest version listed on the
+ * releases page of this repository.
+ *
+ */
+module "adrian_ecr_credentials" {
+  source    = "github.com/ministryofjustice/cloud-platform-terraform-ecr-credentials?ref=2.0"
+  repo_name = "https://github.com/ministryofjustice/cloud-platform-reference-app"
```

Comment:
Repo_name is not the Github URL, but the actual ECR name that will be concatenated as <account-id>.dkr.ecr.eu-west-1.amazonaws.com / team-name / repo-name

Reply:
Ahh, I think I gotcha.